client/my-sites/current-site/notice.jsx

```diff
@@ -244,6 +246,34 @@ export class SiteNotice extends React.Component {
 		return moment( now ).format( format ) === moment( endsAt ).format( format );
 	}
 
+	isEligibleForWhiteGlove() {
+		const createdAt = get( this.props.siteOptions, 'created_at', '' );
+		const HOURS_IN_MS = 60 * 60 * 1000;
+		const isSiteNew = Date.now() - new Date( createdAt ) < 24 * HOURS_IN_MS; // less than 24 hours
+
+		if ( isSiteNew && 'variantShowOffer' === abtest( 'whiteGloveUpsell' ) ) {
+			return true;
+		}
+
+		return false;
+	}
+
+	showWhiteGloveNudge() {
+		const { translate, siteSlug } = this.props;
+		return (
+			<UpsellNudge
+				event={ 'white-glove' }
+				forceHref={ true }
+				callToAction={ translate( 'Upgrade' ) }
+				compact
+				href={ '/checkout/' + siteSlug + '/offer-white-glove' }
+				title={ translate( 'White Glove Offer' ) }
```

Comment:
I'm going to remove the `translate()` for now since we are not anyway waiting for translations, and by the time they come in, we might even have wrapped up the test.